internal/pkg/docker/dockercompose/svc_image.go

```diff
@@ -0,0 +1,144 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/types"
```

Comment:
while in general i avoid this...i wonder if renaming this import would help with readability💭 something like
```suggestion
	compose "github.com/compose-spec/compose-go/types"
```
and then references would look like `compose.BuildConfig` against `manifest.Image`. idk, just a thought

Reply:
Makes sense... I did this and it did improve readability, so I'll push it, thanks for the suggestion!